pkgs/development/tools/mold/default.nix

```
@@ -0,0 +1,38 @@
+{ stdenv
+, fetchFromGitHub
+, lib
+, autoPatchelfHook
+, cmake
+, tbb
+, llvmPackages_latest
+, xxHash
+, zlib
+, openssl
+}:
+
+stdenv.mkDerivation rec {
+  pname = "mold";
+  version = "0.9.2";
+
+  src = fetchFromGitHub {
+    owner = "rui314";
+    repo = pname;
+    rev = "v${version}";
+    sha256 = "sha256-2LXOPirhjAifKYPgngUJwEdGrKMYsRySr5TL2x2p8J0=";
+  };
+
+  buildInputs = [ tbb zlib openssl ];
+  nativeBuildInputs = [ autoPatchelfHook cmake xxHash ];
+
+  dontUseCmakeConfigure = true;
+  EXTRA_LDFLAGS = "--ld-path=${llvmPackages_latest.lld}/bin/ld.lld";
```

```suggestion
  EXTRA_LDFLAGS = "-fuse-ld=${llvmPackages_latest.lld}/bin/ld.lld";
```
Isn't it the proper way to use custom linker?

---

I'm not sure, I just adopted this as a suggestion from @SCOTT-HAMILTON

---

[lld page](https://lld.llvm.org/) also suggests this way.

---

I adopted the suggestion, thanks.